src/osgEarth/MPScreenSpaceLayoutSG

```diff
@@ -56,16 +56,22 @@ public:
                               _id(-1),
                               _originalPriority(0.0),
                               _priority(0.0),
+                              _textPolygonAltitude(0.0),
```

Comment:
Should be _MAX by default as if not defined, but "polygonVisible" defined, we expect the algo to run at all altitude by default

Reply:
actuallly, it is the opposite. CurrentAltitude is from the camera and if maxAlt is equal to 0, it won't use the algorithm.
            
            if (annoDrawable->polygonVisible())
            {
                double maxAlt = annoDrawable->getTextPolygonAltitude();
                if (currentAltitude > maxAlt)
                {
                    visible = true;
                }
                else
                {
                    visible = false;

Reply:
updated as per your suggestion.